locales/en.yml

```diff
@@ -21,27 +41,162 @@ pico:
     exportingBin: "An error occurred while exporting the binary: %s"
     unexportingApp: "An error occurred while unexporting the application: %s"
     unexportingBin: "An error occurred while unexporting the binary: %s"
+  errors:
+    unknownCommand: "Unknown command: %s"
   info:
     initialized: "The Pico subsystem has been initialized."
     exported: "Exported %s\n"
     unexported: "Unexported %s\n"
     exportedApps: "Exported %d applications"
     unexportedApps: "Unexported %d applications"
   export:
-    description: "Export an application or binary from the subsystem."
+    description: "Export an application or binary from the subsystem"
     options:
       app:
-        description: "The name of the application to export."
+        description: "the name of the application to export"
       bin:
-        description: "The name of the binary to export."
+        description: "the name of the binary to export"
       binOutput:
-        description: "Path of the binary output (default: ~/.local/bin/)."
+        description: "path of the binary output (default: ~/.local/bin/)"
   unexport:
-    description: "Unexport an application or binary from the subsystem."
+    description: "Unexport an application or binary from the subsystem"
     options:
       app:
-        description: "The name of the application to unexport."
+        description: "the name of the application to unexport"
       bin:
-        description: "The name of the binary to unexport."
+        description: "the name of the binary to unexport"
       binOutput:
-        description: "Path of the binary output (default: ~/.local/bin/)."
\ No newline at end of file
+        description: "path of the binary output (default: ~/.local/bin/)"
+  init:
+    description: "Initialize the Pico subsystem"
```

Comment:
Agreed, maybe we can use something like this
```suggestion
    description: "Initialize the Vanilla OS Pico subsystem"
```
Or
```suggestion
    description: "Create the Vanilla OS Pico subsystem"
```

Reply:
I think we should keep using "Initialize" since that's inherent to the command, how about:
 ```suggestion
    description: "Initialize the Pico subsystem, used for package management"
```